superset/security/manager.py

```diff
@@ -538,11 +536,10 @@ def create_custom_permissions(self) -> None:
         """
         Create custom FAB permissions.
         """
-
         self.add_permission_view_menu("all_datasource_access", "all_datasource_access")
         self.add_permission_view_menu("all_database_access", "all_database_access")
         self.add_permission_view_menu(
```

Comment:
Why is creating the permission tied to the concept of a `view` and the concept of a `menu`?